core/ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/search/FullTextSearchView.java

```diff
@@ -42,6 +45,17 @@
     /** Set focus to root window. */
     void setFocus();
 
+    /** Returns is accept button in the focus */
+    boolean isAcceptButtonInFocus();
+
+    /** Returns is cancel button in the focus */
```

Comment:
I think that this sentence (and other similar sentences in this PR) grammatically is incorrect.
WDYT about something like:

> Returns true if cancel button is in the focus and false - otherwise.